packages/admin/admin/src/tabs/TabScrollButton.tsx

```diff
@@ -25,20 +27,24 @@ const styles = () => {
             width: "100%",
             height: 40,
         },
-        disabled: {},
+        disabled: {
+            opacity: 0.25,
+        },
     });
 };
 
 function ScrollButton({ orientation, direction, disabled, onClick, classes }: TabScrollButtonProps & WithStyles<typeof styles>) {
+    const deviceType = useCurrentDeviceType();
```

Comment:
Why do we need to do this?

> By default, left and right scroll buttons are automatically presented on desktop and hidden on mobile. (based on viewport width)

https://mui.com/material-ui/react-tabs/#automatic-scroll-buttons

Reply:
Because of this „[...](based on viewport width)“. It should always display buttons on desktop imo. If it's only checked with device width there is a chance that tabs can't be scrolled at all.